parktain/models.py

```diff
@@ -0,0 +1,21 @@
+from sqlalchemy import *
+from sqlalchemy import Column, String, DateTime
+from sqlalchemy.ext.declarative import declarative_base
+
+engine = create_engine('sqlite:///slack-archives.db')
+
+engine.echo = False
+
+Base = declarative_base()
+
+class Message(Base):
+    __tablename__ = 'messages'
+    id = Column(Integer(), primary_key=True, autoincrement=True)
+    user_id = Column(String, nullable=False, index=True)
+    channel_id = Column(String, nullable=False, index=True)
+    message = Column(String(4096))
+    timestamp = Column(DateTime)
+    
+    def __repr__(self):
```

Comment:
It is [convention](https://docs.python.org/2/library/functions.html#repr) for `__repr__` to return a representation that can be eval-ed to get back the same object.  You can override `__str__` instead to be a more readable version.